Makefile.doc

```diff
@@ -209,10 +209,11 @@ install-doc-printable:
 
 install-doc-sphinx:
 	$(MKDIR) $(FULLDOCDIR)/sphinx
-	(for f in `cd doc/sphinx/_build; find . -type f`; do \
-		$(MKDIR) $$(dirname $(FULLDOCDIR)/sphinx/$$f);\
-		$(INSTALLLIB) doc/sphinx/_build/$$f $(FULLDOCDIR)/sphinx/$$f;\
-	done)
+	(for d in html latex; do \
+	for f in `cd doc/sphinx/_build/$$d && find . -type f`; do \
+		$(MKDIR) $$(dirname $(FULLDOCDIR)/sphinx/$$d/$$f);\
+		$(INSTALLLIB) doc/sphinx/_build/$$d/$$f $(FULLDOCDIR)/sphinx/$$d/$$f;\
+	done; done)
```

Comment:
Sorry to be annoying, but why to use another `for`?

I guess you can leave it like this as it is not worth the effort, however the whole rule seem bugged, for example w.r.t. quoting, certainly the `cd` shouldn't be there. Something such as:
```
find doc/sphinx/_build{html,latex} -type f -exec "install -D {} $(FULLDOCDIR)/sphinx/";
```
would be much preferred IMVHO.

[I forgot the exact find syntax]

Reply:
Yeah i'd need a bit more tweaking, never mind, not worth the effort to fix this. The problem of for loops is that you usually need to re-quote.